README.rst

```diff
@@ -10,9 +10,9 @@ PhotoCollage
 
 PhotoCollage allows you to create photo collage posters. It assembles the input
 photographs it is given to generate a big poster. Photos are automatically
-arranged to fill the whole poster, then you can change the final layout,
-dimensions, border or swap photos in the generated grid. Eventually the final
-poster image can be saved in any size.
+arranged to fill the whole poster, then you can swap photos in the generated 
+grid or move them in their cells, and change the final dimensions, border
+style, and output image quality. Eventually the final poster image can be saved.
```

Comment:
I think ` in any size` is important here: can you restore it?

Reply:
I thought "in any size" would be covered by "change the final dimensions", that's why I removed it. I'll change it back.